test/Connect.hs

```diff
@@ -1,17 +1,46 @@
-{-# LANGUAGE CPP, OverloadedStrings #-}
+{-# LANGUAGE CPP               #-}
+{-# LANGUAGE NamedFieldPuns    #-}
+{-# LANGUAGE OverloadedStrings #-}
 module Connect where
 
-import Database.PostgreSQL.Typed (PGDatabase(..), defaultPGDatabase)
-import Network.Socket (SockAddr(SockAddrUnix))
+import           Control.Exception                  (throwIO)
+import qualified Data.ByteString.Char8              as BSC
```

Comment:
test-suite hdbc needs bytestring added to build-depends for this.

Reply:
Indeed, I was developing with `HDBC` disabled and missed it. Fixing it in next commit